app/bundles/PointBundle/Model/PointModel.php

```diff
@@ -251,6 +251,9 @@ public function triggerAction($type, $eventDetails = null, $typeId = null, Lead
                         $delta,
                         $ipAddress
                     );
+                    
+                    $event = new PointActionEvent($action, $lead);
```

Comment:
Shouldn't be the location of the PointActionEvent class defined in a use statement?